kinase-bioactivities-in-chembl/kinase-bioactivities-in-chembl.ipynb

```diff
@@ -0,0 +1,2071 @@
+{
```

Comment:
<p>check the output from this line:</p><p><code>activities['activities.standard_units'].value_counts()</code></p><p>There appear to be some other units involved.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openkinome/kinodata/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>